docs/rfcs/2024-12-06-chat-tags.md

```diff
@@ -0,0 +1,25 @@
+# Chat tags, emoji entry
+
+## Problem
+
+Organize chats.
+
+Stop users from using chat profiles to organize chats.
+
+More fun as we solve groups problem.
+
+## Solution
+
+This is a simple "quality of life" improvement that's long due, as the number of users and conversations grows, that would probably take just a week to ship.
+
+Proposed UX:
+- tags are per profile.
+- preset tags for favourites, contacts, groups and business chats.
+- user defined tags *must* have text and emoji.
```

Comment:
if number of tags + preset tags is bigger than screen width it feels like it should scroll horizontally on mobile devices, but what should we do about desktop? Maybe something similar to user picker where we contain it to X lines and if then allow vertical scroll

Reply:
yes. It does scroll in the code I posted.